org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/webappreader.sh
+finish-args:
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --share=network
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    QTWEBENGINE_DISABLE_SANDBOX: '1'
+modules:
+  - python3-requirements.json
+  - name: webappreader
+    subdir: app
+    buildsystem: simple
+    build-commands:
+      - cp webappreader.py /app/
+      - cp webappreader.sh /app/
+      - chmod +x /app/webappreader.sh
+      - cp constants.py /app/
+      - cp settings.py /app/
+      - cp tools.py /app/
+      - cp tools.py /app/
+      - cp -r ui /app/
+      - cp -r pages /app/
+      - cp -r database /app/
+      - cp -r src /app/
+    sources:
+      - type: dir
+        path: .
+        dest: app
+    post-install:
+      - install -Dm644 org.mintguide.WebAppReader.desktop /app/share/applications/org.mintguide.WebAppReader.desktop
+      - install -Dm644 src/logo.png /app/share/icons/hicolor/128x128/apps/org.mintguide.WebAppReader.png
+      - install -Dm755 wget2 /app/bin/wget2
+      - install -Dm755 libwget.so.2 /app/lib/libwget.so.2
```

Comment:
It's in the runtime

Reply:
in runtime, the wget2 version does not work correctly, there is a bug, so I added a newer one, in the form of a binary, it works normally

Reply:
then build it here

Reply:
build from source?
So the program is accepted here only if the assembly in the manifest comes from the git repository? I can't add, I can't add binaries and libraries?